config/default.yml

```diff
@@ -4015,6 +4015,26 @@ Style/LineEndConcatenation:
   VersionAdded: '0.18'
   VersionChanged: '0.64'
 
+Style/MagicComments:
+  Description: 'Use a consistent style for magic comments.'
+  Enabled: true
+  VersionAdded: '<<next>>'
+  EnforcedStyle: snake
+  SupportedStyles:
+    # `snake` will enforce the magic comment is written
+    # in snake case (words separated by underscores).
+    # Eg: froze_string_literal: true
+    - snake
+    # `kebab` will enforce the magic comment is written
+    # in kebab case (words separated by hyphens).
+    # Eg: froze-string-literal: true
+    - kebab
+  DirectiveCapitalization: lower
```

Comment:
Seems to me this can simply be a boolean.

Reply:
There are three options - `upper` (all uppercase), `lower` (all lowercase) and nil (ignore capitalization)

Reply:
Yeah, I saw this, but was on the fence for whether it makes sense for someone to disable this. I've yet to see someone not writing lowercase magic comments, but I guess they exist. :D

Reply:
I figured that since it's easy, having options is better :smile: